panel/template/fast/css/fast_panel_markdown.css

```diff
@@ -2,11 +2,11 @@
   padding-right: 1em;
   padding-left: 1em;
 }
-.markdown a {
+.main a, .sidebar a {
```

Comment:
I could see links in a HTML pane just had the default blue color. It should also have the accent color.